benchmarks/_example_benchmarks.py

```diff
@@ -79,6 +79,9 @@ class CasesBenchmark(_benchmark.Benchmark):
         ("10", "2"),
     )
 
+    def version_case(self, case: tuple):
+        return 2 * int(case[0]) + int(case[1])
```

Comment:
Am I following correctly that this will end up being 30, 14, and 22?

Reply:
_nods_ we might want to comment that reasoning here to remind our future selves what's going on. My first read of it was that we were making claims about how the version numbers should be derivable from the contents of the case like this, when really they are labels we apply to particular cases.

Reply:
Hm, yeah, maybe for reference purposes it's more useful to put a lookup dict here, e.g. something like 

``` python
case_version_dict = {
    ("10", "10"): 1,
    ("2", "10"): 2,
    ("10", "2"): 1,
}
case_version = case_version_dict[case]
```